### PR TITLE
OHRM5X-1265: Invalidate Cache when Configuring Modules

### PR DESCRIPTION
**Pull request by devishke-orange**
PR contains:
- OHRM5X-1265: [Broken-5beta][Menu- Side Panel] Changes in the side panel is not displayed after enabling/disabling the modules.
- OHRM5X-1271: Reopen issue
- OHRM5X-1277: [Broken][PIM - Memberships] "Required" validation message is not getting displayed when user click save without selecting a membership

### CODE REVIEW

**Comment by devishke-orange** [comment]
**Note**: If admin changes the modules and other users are active and logged in, they will not see the change in the side panel until they logout and log back in. If they try to access disabled modules they will get the "403 Module Forbidden" screen.

**Review by RajithaKumara — APPROVED** [review]

File: src/client/src/core/util/validation/rules.ts [hunk 1]
```diff
@@ -99,6 +99,16 @@ export const digitsOnly = function(value: string): boolean | string {
   );
 };
 
+export const digitsWithDecimalPointOnly = function(
```
Comment (RajithaKumara) [inline src/client/src/core/util/validation/rules.ts:102]: :thinking: 
```suggestion
export const digitsOnlyWithDecimalPoint = function(
```

**Review by RajithaKumara — APPROVED** [review]

File: src/client/src/core/util/validation/rules.ts (hunk 1 above)
Comment (devishke-orange) [reply src/client/src/core/util/validation/rules.ts:102]: yeah this name is better

File: installer/Migration/V5_0_0/lang-string/pim.yaml [hunk 2]
```diff
@@ -512,3 +512,6 @@ langStrings:
   -
     value: 'Terminated on'
     unitId: terminated_on
+  -
+    value: 'Work Email and Other Email cannot be the same'
```
Comment (RajithaKumara) [inline installer/Migration/V5_0_0/lang-string/pim.yaml:516]:
```suggestion
    value: 'Work email and other email cannot be the same'
```
:thinking:

**Pull request merged**